### PR TITLE
Fix tests for storage

### PR DESCRIPTION
**Pull request by ambrussimon**
For E2E-testing flywheel-io/storage-plugins#8, I updated fly/fly's `docker/run-tests.sh` to use core+, and I ran into some errors (unrelated to the referenced plugins PR) and fixed them.

Note that storage-plugins installs a newer pylint in core+, which reveals a bunch of errors. Here I only fixed the minor/obvious little lint problems.

The long-term plan is to upgrade pylint to the same version and fix remaining errors to make the images more coherent and to enable core+ integration testing via fly/fly using non-local storage.

### Review Checklist

- Tests were added to cover all code changes
- Documentation was added / updated
- Code and tests follow standards in CONTRIBUTING.md

### CODE REVIEW

File: tests/integration_tests/python/test_download.py [hunk 1]
```diff
@@ -411,20 +410,21 @@ def test_filelist_range_download(data_builder, as_admin, file_form):
...
-    boundary = r.headers.get('Content-Type').split('boundary=')[1]
-    assert r.content == '--{0}\n' \
-                        'Content-Type: text/csv\n' \
-                        'Content-Range: bytes 1-2/9\n\n' \
-                        '23\n' \
-                        '--{0}\n' \
-                        'Content-Type: text/csv\n' \
-                        'Content-Range: bytes 3-4/9\n\n' \
-                        '45\n'.format(boundary)
+    if os.getenv('SCITRAN_PERSISTENT_FS_URL', 'osfs').startswith('osfs'):
```
Comment (nagem) [inline tests/integration_tests/python/test_download.py]: Why was this changed to only test multiple ranges when using osfs? Is it not supported with other file systems?

**Comment by nagem** [comment]
One small question otherwise LGTM!

File: tests/integration_tests/python/test_download.py (hunk 1 above)
Comment (ambrussimon) [reply tests/integration_tests/python/test_download.py]: Correct, while testing with GS, I got a 400 with an XML body stating that multiple ranges are not supported.
